### PR TITLE
1059 overflow

### PR DESCRIPTION
**Pull request by mjowen**
Fixed issue #1059 by forcing initial condition when _t=0 in the HH analytical simulation
Added a test to ensure this fix doesn't break with the changes in #943 and #571 
The test verifies that:

- An overflow warning is triggered
- No nans are present in the current or the overflown state variable
- The state is correctly solved for tau=0 (starts at initial state then jumps to steady state)

### CODE REVIEW

File: myokit/tests/test_lib_hh.py [hunk 1]
```diff
@@ -914,6 +916,39 @@ def test_against_cvode_v_independent(self):
         e = np.abs(d1['binding.I'] - d2['binding.I'])
         self.assertLess(np.max(e), 2e-4)
 
+    def test_tau_overflow(self):
+        # Overflows leading to tau=0 should still report current
```
Comment (MichaelClerx) [inline myokit/tests/test_lib_hh.py:920]:
```suggestion
        # Overflows leading to tau=0 should still report current
        # https://github.com/myokit/myokit/issues/1059
```

**Review by MichaelClerx — DISMISSED** [review]
Looks good, one minor tweak would be to mention the github issue in the test
Thanks!

**Comment by mjowen** [comment]
By the way, I did check Gary's suggestion of doing indexing on _t!=0 on the first call for each state to avoid the extra np.exp call but the extra lines needed to preallocate the variables and the extra indexing meant it didn't save any time

**Comment by MichaelClerx** [comment]
Afraid merge will have to wait for #1048 and #1062

**Review by MichaelClerx — APPROVED** [review]

**Comment by MichaelClerx** [comment]
Thanks @mjowen !
Will do a release soon so you get this behaviour in the pip installed version

**Pull request merged**